packages/backend/src/nest/connections-manager/connections-manager.service.ts

```diff
@@ -699,7 +699,6 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
       if (this.communityId) {
         this.serverIoProvider.io.emit(SocketActionTypes.COMMUNITY_LAUNCHED, { id: this.communityId })
         this.logger.info('this.libp2pService.connectedPeers', this.libp2pService.connectedPeers)
-        this.logger.info('this.libp2pservice', this.libp2pService)
```

Comment:
This was throwing a circular reference error since under the hood the logger is now doing a `JSON.stringify` on objects.